### ISSUE TITLE
Do I really need "window.opener be gone", if I am using ghacks user.js?

### ISSUE THREAD

**Issue by hunkjazz** [ghacksuserjs/ghacks-user.js#984]
Do I need to install [Firemonkey][2] and [suggested script][1]?

I tried the [test site][3] and it was unable to make changes in opener tab.

Hence, I am assuming `dom.targetBlankNoOpener.enabled` is enough. Although this comment makes me think twice:

> 2429: enable (**limited** but sufficient) window.opener protection [FF65+]

> (**limited** but sufficient)

[1]: https://github.com/ghacksuserjs/ghacks-user.js/wiki/4.2.1-User-Scripts#small_orange_diamond-windowopener-be-gone
[2]: https://addons.mozilla.org/firefox/addon/firemonkey/
[3]: https://mathiasbynens.github.io/rel-noopener/

**Comment by hunkjazz** [comment]
Maybe it's the case of updating this wiki page to include a note, at least?

**Comment by Thorin-Oakenpants** [comment]
Personally I use the extension, as it never fails. @earthlng is the author (of both the script and the extension). My understanding is that the extension covers more cases where it **could** be used (but is probably rare?), but comes with the downside that you can't allow any domain exceptions. Personally, I've never seen an issue, and I think earthlng only ever found microsoft catalog to be problematic: but of course that is all anecdotal.

As for scripts, I think they're problematic and generally risky: also see #891 - which led E to put up the bit in the wiki that it's not reliable in GM

tl;dr: go with the extension and if you hit a problem with a site you sometimes/always use, re-evaluate (or use a different browser for that site). The pref is probably enough for most people

**Comment by hunkjazz** [comment]
> Personally I use the extension, as it never fails.

Does it mean the pref fails sometimes?

> The pref is probably enough for most people

Can you give me a hint on what "enough" means?

**Comment by Thorin-Oakenpants** [comment]
> Does it mean the pref fails sometimes?

No. I meant the script can fail (see #891). The pref shouldn't fail: if it does, then that's a bug

> Can you give me a hint on what "enough" means?

earthlng knows more about it - my understanding is that the pref only covers **common** use cases, so it's **unlikely** that what it doesn't cover will affect you, or at least rarely - but I guess without some real world studies, no-one would know for sure

As to what is not covered by the pref (but is by the extension, not sure about the script) earthlng can explain. As I said, I just use the extension since it doesn't fuck around, covers everything, never fails, and doesn't affect me (sites breaking)

**Comment by rusty-snake** [comment]
https://github.com/ghacksuserjs/ghacks-user.js/issues/401#issuecomment-547868244
